frontend/src/app/forms/forms.page.ts

```diff
@@ -72,6 +73,12 @@ export class FormsPage implements OnInit, OnDestroy {
       field: new FormControl<string[]>([], Validators.required),
       designation: new FormControl<string[]>([], Validators.required),
     }),
+    keyCompetencies: new FormGroup<KeyCompetencies>({
+      languages: new FormControl([], Validators.required),
+      professionalSoftware: new FormControl([]),
+      databases: new FormControl([]),
+      programmingLanguagesAndFrameworks: new FormControl([]),
+    }),
```

Comment:
For now its okay to just use only languages as validator, but maybe we can make this dependent, on which jobs the student is applying to. So as an dev, he should need to fill out the programming languages too.